clarifai/client/auth/helper.py

```diff
@@ -269,9 +273,11 @@ def metadata(self):
       metadata: the metadata need to send with all grpc API calls in the API client.
     """
     if self._pat != "":
-      return (("authorization", "Key %s" % self._pat),)
+      return (("authorization", "Key %s" % self._pat), ("x-clarifai-request-id-prefix",
```

Comment:
`x-clarifai-request-id-prefix` can also be a constant like `REQUEST_ID_PREFIX_HEADER`

Reply:
Done